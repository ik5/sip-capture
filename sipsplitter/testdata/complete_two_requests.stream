INVITE foo@bar SIP/2.0
Content-Length: 0

INVITE bar@baz SIP/2.0
Content-Length: 0

