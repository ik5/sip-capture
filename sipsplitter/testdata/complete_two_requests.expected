{
  "Original": "INVITE foo@bar SIP/2.0\r\nContent-Length: 0\r\n\r\nINVITE bar@baz SIP/2.0\r\nContent-Length: 0\r\n\r\n",
  "Discarded": 0,
  "NoStartLine": 0,
  "StartLine": 2,
  "NoHeaders": 0,
  "Header": 2,
  "NoBody": 0,
  "Body": 2,
  "Complete": 2,
  "Discards": [],
  "Starts": [
    "INVITE foo@bar SIP/2.0\r\n",
    "INVITE bar@baz SIP/2.0\r\n"
  ],
  "Headers": [
    "Content-Length: 0\r\n",
    "Content-Length: 0\r\n"
  ],
  "Bodies": [
    "",
    ""
  ],
  "Messages": [
    "INVITE foo@bar SIP/2.0\r\nContent-Length: 0\r\n\r\n",
    "INVITE bar@baz SIP/2.0\r\nContent-Length: 0\r\n\r\n"
  ]
}